pdsfile/pds3file/rules/JNOSRU_xxxx.py

```
@@ -0,0 +1,105 @@
+             r'previews/\2_thumb.jpg',
+            ]),
+])
+
+##########################################################################################
+# ASSOCIATIONS
+##########################################################################################
+
+associations_to_volumes = translator.TranslatorByRegex([
+    # previews to image files
+    (r'previews/JNOSRU_xxxx(|_v[\d\.]+)/(JNOSRU_0\d\d\d/DATA/SRU_EDR/\w+/\w+/SRU_\d+_20\d{5}T\d{6}_\d\d_V\d\d).*', 0,
+            r'volumes/JNOSRU_xxxx/\2.FIT'),
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)